pallets/pallet-did-lookup/src/account.rs

```diff
@@ -0,0 +1,229 @@
+// Copyright 2019-2022 PureStake Inc.
```

Comment:
I had to copy this in because in the moonbeam repo they are depending on their fork of substrate.

Reply:
@GustavHemmelmayr I would really appreciate your input here!